mlflow/types/utils.py

```diff
@@ -116,7 +116,10 @@ def _infer_schema(data: Any) -> Schema:
             )
         schema = Schema(res)
     elif isinstance(data, pd.Series):
-        schema = Schema([ColSpec(type=_infer_pandas_column(data))])
+        name = None
+        if hasattr(data, "name"):
+            name = data.name
```

Comment:
Nit: I think we can replace this with:

```suggestion
        name = getattr(data, "name", None)
```